### DISCUSSION TITLE
tiles url different 4.9.3 and 4.10.1

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#7465] [Q&A]
Recently, I upgraded itwinjs from version 4.9.3 to 4.10.1.using Minio, I found that the paths for calling tiles before and after the upgrade of itwinjs are different. This will result in the tiles generated in version 4.9.3 being unusable in itwinjs 4.10.1. 
Is there any way to make the previous tiles still usable in itwinjs 4.10.1? Or is there any way to generate all the tiles of a model to Minio at one time?

4.9.3
http://192.168.1.105:10012/modelbucket/afdc3010-e240-4dcf-b99b-245fd9443f8c/tiles/**20_4-E5_0x15**/first/-b-1-0-0-0-1?x-id=GetObject

4.10.1
http://192.168.1.105:10012/modelbucket/afdc3010-e240-4dcf-b99b-245fd9443f8c/tiles/**23_14-E5_0x15**/first/-b-1-0-0-0-1?x-id=GetObject

**Comment by pmconne** [comment]
Hi @paprophet,

Tiles are generated on demand in response to viewing operations, then cached (in your case, in Minio). It is impossible to pregenerate "all" the tiles of an iModel because there are infinite number of them. Also, with each minor release of iTwin.js (like in 4.10), the version of the tile format can change as we make fixes and enhancements. This necessarily requires tiles generated and cached using previous versions of the tile format to be regenerated using the new version of the format.

Please remember that the tile cache is just that - a **cache**, not a pregenerated tileset.
